assets/css/style.css

```diff
@@ -0,0 +1,103 @@
+:root {
+	--prim-color: rgb(73, 3, 165); /* Primary color */
+	--sec-color: rgb(36, 1, 82); /* Secondary color - darker shade of primary */
+	--tert-color: rgb(
+		146,
+		72,
+		204
+	); /* Tertiary color - lighter shade of primary */
+	--quart-color: rgb(182, 141, 217); /* Quaternary color - tint of primary */
+	--font-color: rgb(231, 231, 231);
+	--fady-back-color: rgb(36, 1, 82, 0.2); /* Faded background color */
+}
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **issue (code_clarification):** Consider using a more contrasting font color for better readability.

The current font color is the same as the background color, which will render the text invisible.